### PR TITLE
Rinkeby deployment

### PR DESCRIPTION
**Pull request by marcciosilva**
- Add rinkeby deployments
- Add comments to 11_transfer_sideTokenFactoryToBridge deployment script
- Replace verify.sh with verify.py Python port

### CODE REVIEW

File: bridge/deploy/11_transfer_sideTokenFactoryToBridge.js [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+module.exports = async function({getNamedAccounts, deployments}) { // HardhatRuntimeEnvironment
   const {deployer, sideTokenFactory} = await getNamedAccounts();
   const {log, execute} = deployments;
-
-  if (sideTokenFactory) return
-
-  const BridgeProxy = await deployments.get('BridgeProxy');
-  await execute('SideTokenFactory', {from: deployer}, 'transferPrimary', BridgeProxy.address);
-  log(`SideTokenFactory Transfered Primary to BridgeProxy`);
+  if (sideTokenFactory) {
```
Comment (marcciosilva) [inline bridge/deploy/11_transfer_sideTokenFactoryToBridge.js:16]: sonarcloud stuff

File: bridge/hardhat.config.js [hunk 2]
```diff
@@ -192,61 +206,80 @@ function getNamedAccounts() {
     allowTokensProxy: getAllowTokensProxyAddressesByChainId(),
     bridgeProxy: getBridgeProxyAddressesByChainId(),
     federatorProxy: getFederatorProxyAddressesByChainId(),
+    sideTokenFactory: getSideTokenFactoryAddressesByChainId(),
   };
 }
 
 function getMultiSigAddressesByChainId() {
-  let multiSigAddressesByChainId = {};
+  const multiSigAddressesByChainId = {};
```
Comment (marcciosilva) [inline bridge/hardhat.config.js:214]: declaring dictionaries/maps as `const` as per sonarcloud suggestion

File: bridge/hardhat/helper/chains.js [hunk 3]
```diff
@@ -28,7 +29,7 @@ function tokenSymbol(network) {
   return network.config.token_symbol ?? 'e';
 }
 
-function isMainet(network) {
+function isMainnet(network) {
```
Comment (marcciosilva) [inline bridge/hardhat/helper/chains.js:32]: typo

**Review by pmprete — APPROVED** [review]
Aswome work!!

**Pull request merged**